Sources/GRPC/ClientConnection.swift

```diff
@@ -312,6 +312,9 @@ extension ClientConnection {
       return self.tls == nil ? .http : .https
     }
 
+    /// A custom verification callback that allows completely overriding the certificate verification logic for this connection.
+    public var customVerificationCallback: NIOSSLCustomVerificationCallback?
```

Comment:
Since this relates to TLS it should be on the `ClientConnection.Configuration.TLS` struct rather than the general configuration one.